testing/trino-testing-containers/src/main/java/io/trino/testing/containers/Alluxio.java

```diff
@@ -43,8 +47,16 @@ public Alluxio()
 
     public void start()
     {
-        alluxioMaster.start();
-        alluxioWorker.start();
+        Failsafe.with(RetryPolicy.<Void>builder()
+                        .withMaxAttempts(10)
+                        .withBackoff(100, 500, MILLIS)
```

Comment:
max delay of 0.5 is very short for container that starts in seconds-minutes. 

```suggestion
.withBackoff(1, 15, SECONDS)
```